File: app/src/main/java/com/example/android/popularmovies/utils/ImageUtils.java
package com.example.android.popularmovies.utils;

import android.databinding.BindingAdapter;
import android.widget.ImageView;

import com.bumptech.glide.Glide;

public class ImageUtils {

    @BindingAdapter("loadImage")
    public static void loadImage(ImageView imageView, String url) {
        Glide.with(imageView.getContext()).load(url).into(imageView);
    }
}

File: app/src/main/kotlin/com/example/android/popularmovies/utils/ImageUtils.kt
package com.example.android.popularmovies.utils

import androidx.databinding.BindingAdapter
import android.widget.ImageView

import com.bumptech.glide.Glide

object ImageUtils {

    @BindingAdapter("loadImage")
    @JvmStatic
    fun loadImage(imageView: ImageView, url: String) {
        Glide.with(imageView.context).load(url).into(imageView)
    }
}
